content/developer/Vardefs.adoc

```diff
@@ -327,3 +329,69 @@ existing one e.g changing a field type:
 .Example 4.6: Example overriding an existing vardef
 [source,php]
 $dictionary["TheModule"]["fields"]["some_field"]['type'] = 'int';
+
+
+== Frontend
+
+When developing some complex JS functionality is very useful to have module vardefs definitions or custom data in the DOM so you can read it from JS.
```

Comment:
Please change a few of the times you write `JS` to `JavaScript` so that people searching for JavaScript can find this article.